akvo/rsr/models/organisation.py

```diff
@@ -321,7 +321,31 @@ def content_owned_organisations(self):
             return queryset
 
     def __unicode__(self):
-        return self.name
+        return self.get_name
+
+    @property
+    def get_name(self):
```

Comment:
Ya, ideally it should be "name"...already taken 😛  That's why I settled for get_name.